### PR TITLE
add imge folder and html file and css file

### LINKED ISSUE #6: About US
## Description of the working group

*  who are we ?
*  What we do ?

### PR DESCRIPTION
**Pull request by abdullahMuzyed**
relates #6

### CODE REVIEW

**Review by mo7mad4 — DISMISSED** [review]
cool

**Review by Tuqa-yasin — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -4,9 +4,48 @@
...
+            </div>
+
+            </div>
+
+        </div>
+        </div>
+
+    </div>
+=======
     <title>Document</title>
```
Comment (AlaaTaima) [inline index.html]: **remove these lines**

File: style.css [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+.about_us{
+    padding: 40px 0px;
+    background-color: #dff4f5;
+    height: 840px;  
+    
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 
```suggestion
.about_us{
    padding: 40px 0px;
    background-color: rgb(223, 244, 245);
    height: 126vh;
    
}
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+.container{
+    width: 100%;
+    padding-right:15px ;
+    padding-left: 15px;
+    margin-right: auto;
+    margin-left: auto;
+    
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 

```suggestion
.container{
       display: flex;
    justify-content: center;
    align-items: center;
    
}
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    height: 410px;
+}
+.border_second_paragraf{
+    width: 481px;
+    height: 410px;
+}
+.border{
+    
+    margin-left: 268px;
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 
```suggestion
.border{
    
        display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
}
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    
+}
+
+.About_A{
+    text-align: center;
+    padding-bottom:50px ;
+    margin-top: 40px;
+    width: 960px;
+    height: 186px;
+}
```
Comment (AlaaTaima) [inline style.css]: update this style as the following : 
```suggestion
.About_A{
    text-align: center;
    width: 100%;
    height: 27vh;
}
```

File: style.css [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+#Us{
+    color: black;
+}
+.first_paragraph{
+    color: #141012a9;
+    font-size: 17px;
+    line-height: 32px;
+    text-align: center;
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 
```suggestion
.first_paragraph{
        color: rgba(20, 16, 18, 0.663);
    font-size: 17px;
    line-height: 32px;
    text-align: center;
    width: 80%;
    margin: 0 auto;
}
```

File: style.css [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+.About_content{
+    border: 2px solid #1687a7;
+    border-radius: 25px;
+    padding: 40px 40px ;
+    height: 494px;
+    width: 960px;
+    display: flex;
+    justify-content: center;
+
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 

```suggestion
.About_content{
   border: 2px solid rgb(22, 135, 167);
    border-radius: 25px;
    padding: 40px;
    height: 80vh;
    width: 90%;
    display: flex;
    justify-content: space-around;
    align-items: center
    }
```

File: style.css [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+.second_paragraph{
+    
+    width: 481px;
+    height: 320px;
+    font-size: 15px;
+    line-height: 32px;
+    color: #0000008a;
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 

```suggestion
.second_paragraph{
    
 text-align: justify;
    font-size: 17px;
    line-height: 35px;
    color: rgba(0, 0, 0, 0.54);
}
```

File: style.css [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+a:hover{
+    background-color: #1687a7;
+}
+
+.imge{
+    width: 335px;
+    height: 374px;
+    padding-top: 30px;
+    padding-left: 20px;
+}
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 
```suggestion
.imge{
  
    padding-left: 20px;
}
```

File: style.css [hunk 10]
```diff
@@ -0,0 +1,113 @@
...
+
+img{
+    width: 321px;
+    height: 390px;
+    border-radius: 3px #f9f3f3;
```
Comment (AlaaTaima) [inline style.css]: **update this style as the following :** 

```suggestion
img{
width: 321px;
    height: 389px;
    border: 2px solid #fdefe4;
    }
```

**Pull request merged**